coralproject/talk
# Update text and description in moderation configure view

## What does this PR do?

Issue #1920 

## How do I test this PR?

- Navigate to Moderation View > Configure
- Check it all looks 👍
